sources/europe/de/Worms-2003.geojson

```diff
@@ -0,0 +1,29 @@
+{
+    "type": "Feature",
+    "properties": {
+        "name": "Worms 2003",
+        "type": "wms",
+        "url": "https://geoportal-worms.de/ogc/wms/luftbild-2003?LAYERS=D763F3C28EC14D8BAB4C307D33306FAF&STYLES=&FORMAT=image/png&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap",
+        "license_url": "https://wiki.openstreetmap.org/wiki/DE:Permissions/Luftbild_Worms",

```

Comment:
I'm not really in a position to assess the legalities here, but this sounds like a clear okay that they allow derived data like traced features to be owned by you, just we can't publish the actual imagery.

Reply:
@andrewharvey Yeah, I wrote the city myself and, as described on the Wiki page, they clearly stated, derived features may be used.